src/main/java/pro/sky/homework17/controller/LoginController.java

```diff
@@ -0,0 +1,38 @@
+package pro.sky.homework17.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import pro.sky.homework17.exception.WrongLoginException;
+import pro.sky.homework17.exception.WrongPasswordException;
+
+@RestController
+public class LoginController {
+    @GetMapping(path = "/login")
+    public static void Login(@RequestParam("login") String login,
```

Comment:
```suggestion
    public static void login(@RequestParam("login") String login,
```
названия методов надо писать с маленькой буквы